[2022-01-16 01:41:03] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Target [Illuminate\Contracts\Http\Kernel] is not instantiable. in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1093
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(886): Illuminate\Container\Container->notInstantiable('Illuminate\\Cont...')
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\Cont...')
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array)
#3 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\Application.php(300): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#4 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\helpers.php(42): Laravel\Lumen\Application->make('Illuminate\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\Cont...')
#6 {main}
  throw {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Target [Illuminate\\Contracts\\Http\\Kernel] is not instantiable. in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(886): Illuminate\\Container\\Container->notInstantiable('Illuminate\\\\Cont...')
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\Cont...')
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cont...', Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\Application.php(300): Illuminate\\Container\\Container->make('Illuminate\\\\Cont...', Array)
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\helpers.php(42): Laravel\\Lumen\\Application->make('Illuminate\\\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\\\Cont...')
#6 {main}
  throw at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093)
[stacktrace]
#0 {main}
"} 
[2022-01-16 01:41:03] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 01:41:03] local.ERROR: Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 01:41:04] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:08:02] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Target [Illuminate\Contracts\Http\Kernel] is not instantiable. in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1093
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(886): Illuminate\Container\Container->notInstantiable('Illuminate\\Cont...')
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\Cont...')
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array)
#3 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\Application.php(300): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#4 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\helpers.php(42): Laravel\Lumen\Application->make('Illuminate\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\Cont...')
#6 {main}
  throw {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Target [Illuminate\\Contracts\\Http\\Kernel] is not instantiable. in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(886): Illuminate\\Container\\Container->notInstantiable('Illuminate\\\\Cont...')
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\Cont...')
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cont...', Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\Application.php(300): Illuminate\\Container\\Container->make('Illuminate\\\\Cont...', Array)
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\helpers.php(42): Laravel\\Lumen\\Application->make('Illuminate\\\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\\\Cont...')
#6 {main}
  throw at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:08:02] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:08:02] local.ERROR: Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:08:02] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:08:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:10:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:11:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:11:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:12:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:13:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:14:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:16:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:16:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:18:00] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:18:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:19:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:19:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:21:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:21:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:22:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:23:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:24:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:25:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:26:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:27:59] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:28:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:28:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:30:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:30:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:31:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:33:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:33:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:35:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:36:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:37:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:37:59] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:37:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:38:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:39:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:41:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:42:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:43:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:43:59] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:45:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:46:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:47:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:47:59] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:48:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:49:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:50:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:51:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:52:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:53:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:54:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:55:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:56:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:57:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:57:59] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:58:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 04:59:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:00:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:01:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:02:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:03:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:04:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:05:03] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:06:04] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:07:03] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:08:01] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:08:01] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:09:05] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:10:01] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:11:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:12:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:13:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:14:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:15:01] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:16:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 05:17:00] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:45:21] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:45:21] local.ERROR: Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to a member function getRoutes() on array in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:45:21] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Target [Illuminate\Contracts\Http\Kernel] is not instantiable. in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1093
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(886): Illuminate\Container\Container->notInstantiable('Illuminate\\Cont...')
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\Cont...')
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array)
#3 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\Application.php(300): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#4 E:\xampp\htdocs\3class\API\vendor\laravel\lumen-framework\src\helpers.php(42): Laravel\Lumen\Application->make('Illuminate\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\Cont...')
#6 {main}
  throw {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Target [Illuminate\\Contracts\\Http\\Kernel] is not instantiable. in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(886): Illuminate\\Container\\Container->notInstantiable('Illuminate\\\\Cont...')
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\Cont...')
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cont...', Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\Application.php(300): Illuminate\\Container\\Container->make('Illuminate\\\\Cont...', Array)
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\helpers.php(42): Laravel\\Lumen\\Application->make('Illuminate\\\\Cont...', Array)
#5 Command line code(1): app('Illuminate\\\\Cont...')
#6 {main}
  throw at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1093)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:45:21] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:46:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:47:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:47:57] local.ERROR: Command "sá»" is not defined. {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"sá»\" is not defined. at E:\\xampp\\htdocs\\3class\\API\\vendor\\symfony\\console\\Application.php:709)
[stacktrace]
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\symfony\\console\\Application.php(259): Symfony\\Component\\Console\\Application->find('s\\xC3\\xA1\\xC2\\xBB\\xC2\\x83')
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\symfony\\console\\Application.php(171): Symfony\\Component\\Console\\Application->doRun(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\console\\Application.php(94): Symfony\\Component\\Console\\Application->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\laravel\\lumen-framework\\src\\Console\\Kernel.php(116): Illuminate\\Console\\Application->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#4 E:\\xampp\\htdocs\\3class\\API\\artisan(35): Laravel\\Lumen\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#5 {main}
"} 
[2022-01-16 11:48:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:49:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:50:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:51:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:52:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:53:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:54:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:55:20] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:55:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:56:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:57:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:58:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 11:59:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:00:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:01:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:02:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:03:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:04:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:05:20] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:05:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:06:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:07:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:08:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:09:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:10:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:11:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:12:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:13:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:14:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:15:20] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:15:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:16:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:17:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:18:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:19:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:20:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:21:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:22:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:23:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:24:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:25:20] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:25:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:26:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:27:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:28:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:29:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:30:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:31:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:32:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:33:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:34:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:35:20] local.ERROR: Uncaught Illuminate\Contracts\Container\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\View\Compilers\Compiler in E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php:1108
Stack trace:
#0 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(1015): Illuminate\Container\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(950): Illuminate\Container\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(912): Illuminate\Container\Container->resolveDependencies(Array)
#3 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(758): Illuminate\Container\Container->build('Illuminate\\View...')
#4 E:\xampp\htdocs\3class\API\vendor\illuminate\container\Container.php(694): Illuminate\Container\Container->resolve('Illuminate\\View...', Array)
#5 E: {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Illuminate\\Contracts\\Container\\BindingResolutionException: Unresolvable dependency resolving [Parameter #1 [ <required> $cachePath ]] in class Illuminate\\View\\Compilers\\Compiler in E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108
Stack trace:
#0 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(1015): Illuminate\\Container\\Container->unresolvablePrimitive(Object(ReflectionParameter))
#1 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(950): Illuminate\\Container\\Container->resolvePrimitive(Object(ReflectionParameter))
#2 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(912): Illuminate\\Container\\Container->resolveDependencies(Array)
#3 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(758): Illuminate\\Container\\Container->build('Illuminate\\\\View...')
#4 E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php(694): Illuminate\\Container\\Container->resolve('Illuminate\\\\View...', Array)
#5 E: at E:\\xampp\\htdocs\\3class\\API\\vendor\\illuminate\\container\\Container.php:1108)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:35:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:36:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:37:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:38:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:39:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:40:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:41:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
[2022-01-16 12:42:20] local.ERROR: Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Call to undefined function app_path() in Command line code:1
Stack trace:
#0 {main}
  thrown at Command line code:1)
[stacktrace]
#0 {main}
"} 
